### PR TITLE
AVFoundationPlayer dance backwards - fixes

### PR DESCRIPTION
**Pull request by i-n-g-o**
as explained here:
https://github.com/openframeworks/openFrameworks/issues/3629

it also fixes frame-stepping issues for pre OSX 10.8 and improves accuracy when seeking for time

### CODE REVIEW

**Comment by i-n-g-o** [comment]
hm.
for pre OSX 10.8 the old code is able to step every frame. it has the problem though that it does not start at the first frame.
i'll investigate...

**Comment by i-n-g-o** [comment]
ok. this was just a matter of precision.
all good now :)

File: apps/projectGenerator [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit b5fafc830128915cef181ae5c1cb76fbf75a9aee
```
Comment (ofTheo) [inline apps/projectGenerator]: should this change be included in the PR?

File: libs/openFrameworksCompiled/project/osx/openFrameworksLib.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -188,7 +188,7 @@
...
 		6766729E1A749D1900400051 /* ofAVFoundationPlayer.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = ofAVFoundationPlayer.h; sourceTree = "<group>"; };
 		6766729F1A749D1900400051 /* ofAVFoundationVideoPlayer.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = ofAVFoundationVideoPlayer.h; sourceTree = "<group>"; };
-		676672A21A749D1900400051 /* ofAVFoundationPlayer.mm */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.objcpp; path = ofAVFoundationPlayer.mm; sourceTree = "<group>"; };
+		676672A21A749D1900400051 /* ofAVFoundationPlayer.mm */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.objcpp; path = ofAVFoundationPlayer.mm; sourceTree = "<group>"; usesTabs = 0; };
 		67D96B941651AF6D00D5242D /* ofGLUtils.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; name = ofGLUtils.cpp; path = gl/ofGLUtils.cpp; sourceTree = "<group>"; };
```
Comment (ofTheo) [inline libs/openFrameworksCompiled/project/osx/openFrameworksLib.xcodeproj/project.pbxproj]: also prob shouldn't be committing this file either  :)

**Comment by ofTheo** [comment]
Thanks so much @i-n-g-o ! 
just noted a couple of small files that shouldn't be committed with this PR. 

So with this PR does the videoPlayerExample now work as it used to with backwards playback and scrubbing working well?

**Comment by i-n-g-o** [comment]
removed some stuff in seekToTime:withTolerance:
stuff i thought would help improve accuracy, but it was the float/double thing anyways.

strange for the additional files in this PR... should i revert? or is it fine like this?

i did not test iOS...
would be great if somebody fluent with iOS could do some testing...

**Comment by ofTheo** [comment]
ping @danoli3 to maybe give a quick test on iOS

**Comment by ofTheo** [comment]
@i-n-g-o if you can remove those files from being committed without redoing the PR that would be ideal

File: apps/projectGenerator (hunk 1 above)
Comment (bilderbuchi) [inline apps/projectGenerator]: I'm guessing this comes from the git submodule sync - probably someone committed something to the PG, and forgot to update the submodule within OF, so this gets done now.

**Comment by i-n-g-o** [comment]
hm. tried to change branching-point with `git branch -f`
not sure, if this resolves the issues...

i could also ditch this and create a new PR

**Comment by ofTheo** [comment]
looks good to me. 
I only see the three files relating to the AV Video stuff. 
Thanks @i-n-g-o

**Comment by i-n-g-o** [comment]
cool

**Pull request merged**